components/layout/display_list_builder.rs

```diff
@@ -2262,16 +2261,19 @@ impl FragmentDisplayListBuilding for Fragment {
     }
 }
 
-#[derive(Clone, Copy, PartialEq)]
-pub enum EstablishContainingBlock {
-    Yes,
-    No,
+bitflags! {
+    pub flags StackingContextCollectionFlags: u8 {
+        #[doc = "This flow never establishes a containing block."]
```

Comment:
nit: You can use `///` style comments, if you prefer.

Reply:
Okay. I'll do this. Thanks for the review!